pkg/netpol/connlist/connlist.go

```diff
@@ -371,16 +374,39 @@ func (ca *ConnlistAnalyzer) getConnectionsList(pe *eval.PolicyEngine, ia *ingres
 		return connsRes, []Peer{}, nil
 	}
 
+	// get workload peers and ip blocks
+	peerList, err := pe.GetPeersList()
+	if err != nil {
+		ca.errors = append(ca.errors, newResourceEvaluationError(err))
+		return nil, nil, err
+	}
```

Comment:
convert right after this the `peerList` to be of type `[]connlist.Peer` instead of `[]eval.Peer` , and use all functions on this pkg with arguments of type `Peer` instead of `eval.Peer` consistently

Reply:
done